app-mini/build.gradle

```diff
@@ -100,8 +100,8 @@ android {
         exclude 'lib/arm64-v8a/obfs4proxy.so'
         exclude 'lib/armeabi-v7a/obfs4proxy.so'
         exclude 'lib/armeabi/obfs4proxy.so'
+        exclude 'META-INF/androidx.localbroadcastmanager_localbroadcastmanager.version'
```

Comment:
is this necessary?

Reply:
last I checked it was required because there was a conflict